### ISSUE TITLE
Bedtools merge fails

### ISSUE THREAD

**Issue by Emiller88** [hyulab/PINTS#12]
With a small test dataset, but it doesn't fial for homer/grohmm

## Steps to reproduce

1. Install nextflow `curl -s https://get.nextflow.io | bash`
2. `NXF_VER=21.10.3 ./nextflow run nf-core/nascent -r pints -profile test,docker --outdir results`

- [.command.log](https://github.com/hyulab/PINTS/files/9522096/default.command.log)
- [peakcalling_2022_09_07_23_03_22_29.log](https://github.com/hyulab/PINTS/files/9522097/peakcalling_2022_09_07_23_03_22_29.log)
- [CI of latest commit](https://github.com/nf-core/nascent/actions/runs/3011335022)

**Comment by liyao001** [comment]
Thanks a lot for incorporating PINTS to the nascent transcription processing pipeline! I'll try to reproduce this issue, and report back.

**Comment by liyao001** [comment]
Sorry for the delay! I reproduced the error with the inputs you provided. This error happened when PINTS tried to merge all predictions while specific type(s) of predictions (bidirectional/divergent/unidirectional peaks) were empty. I fixed this problem in the recent commit, and I'll post updates to [nf-core/nascent#39](https://github.com/nf-core/nascent/issues/39) when the new release is reflected on the bioconda channel. Again, thanks a lot for helping us spot this bug!
